src/libutils/oglapphelpers/glsl.cpp

```diff
@@ -452,6 +452,14 @@ std::string OpenGLBuilder::getGLSLVersionString()
     {
         return "#version 400 core";
     }
+    else if (m_shaderDesc->getLanguage() == GPU_LANGUAGE_GLSL_ES_1_0)
```

Comment:
[Not from your branch] Please replace the `ifs` by a `switch` so a new enum value generates a build break i.e. easier to see where to add something.